airflow/www/templates/airflow/dag_code.html

```diff
@@ -21,6 +21,14 @@
 
 {% block page_title %}{{ dag.dag_id }} - Code - Airflow{% endblock %}
 
+{% block head %}
+  {{ super() }}
+  <head>
+    <meta name="demo_mode" content="{{ demo_mode }}">
+    <meta name="wrapped" content="{{ wrapped }}">
+  </head>
```

Comment:
I don't _think_ there is a `head` block, but instead `head_meta`, `head_js`, or `head_css`. The `<head></head>` tags will already be provided by the the parent template.
```suggestion
{% block head_meta %}
  {{ super() }}
  <meta name="demo_mode" content="{{ demo_mode }}">
  <meta name="wrapped" content="{{ wrapped }}">
```

Reply:
https://github.com/dpgaspar/Flask-AppBuilder/blob/master/flask_appbuilder/templates/appbuilder/init.html is the template that is providing most of the "base" HTML